arcgis-ios-sdk-samples/Route & Directions/Find closest facility to multiple incidents (service)/FindClosestFacilityMultipleIncidentsServiceViewController.swift

```diff
@@ -0,0 +1,226 @@
+    
+    /// The graphics overlay for the routes.
+    let routesOverlay = AGSGraphicsOverlay()
+    
+    /// The facility features.
+    var facilityFeatures = [AGSFeature]()
+    /// The incident features.
+    var incidentFeatures = [AGSFeature]()
+    /// The task used to find the closest facilities.
+    let closestFacilityTask: AGSClosestFacilityTask = {
+        let url = URL(string: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/NetworkAnalysis/SanDiego/NAServer/ClosestFacility")!
+       return AGSClosestFacilityTask(url: url)
+    }()
+    
+    /// Creates a feature layer with the facilities. It is configured to render
+    /// the facilities using a fire station image.
+    ///
+    /// - Returns: A new `AGSFeatureLayer` object.
+    func makeFacilitiesLayer() -> AGSFeatureLayer {
+        let facilitiesTableURL = URL(string: "https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Facilities/FeatureServer/0")!
+        let facilitiesTable = AGSServiceFeatureTable(url: facilitiesTableURL)
+        let facilitiesLayer = AGSFeatureLayer(featureTable: facilitiesTable)
+        
+        let facilityImageURL = URL(string: "https://static.arcgis.com/images/Symbols/SafetyHealth/FireStation.png")!
+        let facilitySymbol = AGSPictureMarkerSymbol(url: facilityImageURL)
+        facilitySymbol.width = 30
+        facilitySymbol.height = 30
+        facilitiesLayer.renderer = AGSSimpleRenderer(symbol: facilitySymbol)
+        
+        return facilitiesLayer
+    }
+    
+    /// Creates a feature layer with the incidents. It is configured to render
+    /// the incidents using a fire image.
+    ///
+    /// - Returns: A new `AGSFeatureLayer` object.
+    func makeIncidentsLayer() -> AGSFeatureLayer {
+        let incidentsTableURL = URL(string: "https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Incidents/FeatureServer/0")!
+        let incidentsTable = AGSServiceFeatureTable(url: incidentsTableURL)
+        let incidentsLayer = AGSFeatureLayer(featureTable: incidentsTable)
+        
+        let incidentsImageURL = URL(string: "https://static.arcgis.com/images/Symbols/SafetyHealth/esriCrimeMarker_56_Gradient.png")!
+        let incidentsSymbol = AGSPictureMarkerSymbol(url: incidentsImageURL)
+        incidentsSymbol.width = 30
+        incidentsSymbol.height = 30
+        incidentsLayer.renderer = AGSSimpleRenderer(symbol: incidentsSymbol)
+        
+        return incidentsLayer
+    }
+    
+    /// Creates a map.
+    ///
+    /// - Returns: A new `AGSMap` object.
+    func makeMap() -> AGSMap {
+        let map = AGSMap(basemap: .streetsWithReliefVector())
+        
+        let facilitiesLayer = makeFacilitiesLayer()
+        let incidentsLayer = makeIncidentsLayer()
+        map.operationalLayers.addObjects(from: [facilitiesLayer, incidentsLayer])
+        
+        let dispatchGroup = DispatchGroup()
+        dispatchGroup.enter()
+        dispatchGroup.enter()
```

Comment:
move this to line 106

Reply:
I doesn't make a difference either way. But for you, I'll move it.